src/openms/include/OpenMS/ANALYSIS/ID/IDScoreGetterSetter.h

```diff
@@ -240,14 +240,76 @@ namespace OpenMS
      * @brief Helper for getting scores in ConsensusMaps
      * @todo allow FeatureMap?
      */
-    template<class ...Args>
     static void getPeptideScoresFromMap_(
         ScoreToTgtDecLabelPairs &scores_labels,
-        const ConsensusMap &cmap, bool include_unassigned_peptides, Args &&... args)
+        const ConsensusMap &cmap, bool include_unassigned_peptides)
```

Comment:
tag here?
`// GCC-OPT 4.8 -- can be replaced by a variadic template, see #4273`